File: be/src/main/java/com/project/popupmarket/entity/StagingPayment.java
package com.project.popupmarket.entity;

import lombok.*;

import java.io.Serializable;
import java.math.BigDecimal;
import java.time.LocalDate;
import java.time.LocalDateTime;

@Getter
@Builder
@ToString
@NoArgsConstructor
@AllArgsConstructor
public class StagingPayment implements Serializable {
    private String orderId;
    private Long customerId;
    private Long rentalLandId;
    private LocalDate startDate;
    private LocalDate endDate;
    private BigDecimal totalAmount;
}

File: be/src/main/java/com/project/popupmarket/entity/StagingPayment.kt
package com.project.popupmarket.entity

import java.io.Serializable
import java.math.BigDecimal
import java.time.LocalDate

data class StagingPayment(
    val orderId: String,
    val customerId: Long,
    val rentalLandId: Long,
    val startDate: LocalDate,
    val endDate: LocalDate,
    val totalAmount: BigDecimal,
) : Serializable